.gitmodules

```diff
@@ -1,9 +1,9 @@
-[submodule "third_party/coqutil"]
-	path = third_party/coqutil
-	url = https://github.com/mit-plv/coqutil
```

Comment:
I guess the preference of the bedrock coqutil version is that it simpler to depend on that version as we don't have to mess with the bedrock submodule?

Reply:
Pretty much. We don't want a situation where, for instance, we update our coqutil version and bedrock2 hasn't yet updated theirs. Then we would have two different definitions in scope for everything in coqutil -- bedrock2 would be using one kind of coqutil `map` and we would be using another. And in order to update bedrock2 to match our version of coqutil, we might have to do nontrivial work on bedrock2 to update it, which we probably want to avoid.